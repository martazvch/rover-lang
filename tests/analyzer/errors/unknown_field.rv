struct Person {
    age: int,
}

fn main() {
    var tom = Person.{agge = 4}
}

