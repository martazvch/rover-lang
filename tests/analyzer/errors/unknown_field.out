unknown_struct_field, agge
